doc/projects.md

```diff
@@ -90,6 +90,7 @@ Option           | Documentation
 :test-dir        | A path, relative to the project root, where the test code lives.
 :test-prefix     | A prefix to generate test files names.
 :test-suffix     | A suffix to generate test files names.
+:related-file    | A function to specify test/impl/other files in a more flexible way.
```

Comment:
I'd name this `related-files` or `related-files-fn`. In hinsight I regret not adding a `-fn` suffix to the keys that are functions.

Reply:
Sounds good. I will change as suggested.